505.xml

```diff
@@ -0,0 +1,265 @@
+1 - CoAPS (as defined in RFC 7252) with the additional support for block-wise transfer 
+2 - HTTP 1.1 (as defined in RFC 7230) 
+3 - HTTPS 1.1 (as defined in RFC 7230) Additional values MAY be defined in the future. Any value not understood by the LwM2M Server MUST be ignored.]]></Description>
+			</Item>
+			<Item ID="11">
+				<Name>Profile Update Delivery Method</Name>
+				<Operations>R</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Mandatory</Mandatory>
+				<Type>Integer</Type>
+				<RangeEnumeration></RangeEnumeration>
+				<Units></Units>
+				<Description><![CDATA[The LwM2M Client uses this resource to
+indicate its support for transferring Profile
+images to the client either via the Package
+Resource (=push) or via the Package URI
+Resource (=pull) mechanism.
+0 - Pull only
+1 - Push only
+2 - Both. In this case the LwM2M Server
+MAY choose the preferred mechanism for
+conveying the profile image to the
+LwM2M Client.]]></Description>
+			</Item>
+			<Item ID="12">
+				<Name>Free Memory on SIM</Name>
+				<Operations>R</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>Integer</Type>
+				<RangeEnumeration></RangeEnumeration>
+				<Units>bit</Units>
+				<Description><![CDATA[Estimated current available amount of storage space on SIM which can store data and software in the LwM2M Device (expressed in kilobytes).]]></Description>
+			</Item>
+			<Item ID="13">
+				<Name>Total Memory on SIM</Name>
+				<Operations>R</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>Integer</Type>
+				<RangeEnumeration></RangeEnumeration>
+				<Units>bit</Units>
+				<Description><![CDATA[Total amount of storage space which can store data and software in the LwM2M Device]]></Description>
+			</Item>
+			<Item ID="14">
+				<Name>Integrated Circuit Card Identifier (ICCID)</Name>
+				<Operations>R</Operations>
+				<MultipleInstances>Multiple</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>Integer</Type>
+				<RangeEnumeration></RangeEnumeration>
+				<Units></Units>
+				<Description><![CDATA[This EF provides a unique identification number for the UICC/Smart Cards. Please refer ETSI TS 102.22.1]]></Description>
+			</Item>
+			<Item ID="15">
+				<Name>eUICC ID</Name>
+				<Operations>R</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>String</Type>
+				<RangeEnumeration></RangeEnumeration>
+				<Units></Units>
+				<Description><![CDATA[eUICC-ID (a.k.a. EID), see GSMA SGP.02 and GSMA SGP.29 for definitions]]></Description>
+			</Item>
+			<Item ID="16">
+				<Name>Profile Type</Name>
+				<Operations>R</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>Integer</Type>
```

Comment:
This resource has type Integer, but is this really accurate? In SGP.22 it is specified as a string and also in TrustedConnectivityAlliance specification "eUICC Profile Package: Interoperable Format Technical Specification".

Reply:
Change to "String"